### PR TITLE
GO-11821 fix crash when click customize share icon from iPad

### PR DESCRIPTION
**Pull request by shupingchu**
Jira: https://servicemax.atlassian.net/browse/GO-11821
Video

iPhone X:
RecordIt-1585846777-iphone
 
iPad Pro 12 - screenshot (video file too larget  to upload)
ED2D1EFD-C31E-460F-9443-83BD98DB1C78

FC3FEC1F-58DA-4063-87E5-8F3F53FC3D83

### CODE REVIEW

File: ios/QLPreviewControllerCustom.m [hunk 1]
```diff
@@ -48,6 +48,19 @@ - (void)shareAction {
                        animated:YES
                      completion:^{}];
     
+    //for iPhone
```
Comment (jiwansvmx) [inline ios/QLPreviewControllerCustom.m:51]: Don't you have to get rid of line 47 ?

**Comment by jiwansvmx** [comment]
Also please attach either a screenshot or a gif after you changes ?

File: ios/QLPreviewControllerCustom.m (hunk 1 above)
Comment (shupingchu) [reply ios/QLPreviewControllerCustom.m:51]: Oh you right!! Thank you for point it out, very important. Sorry forgot to remove it. And will bring the short vide as well.

Comment (shupingchu) [reply ios/QLPreviewControllerCustom.m:47]: Video Updated!!

**Comment by shupingchu** [comment]
Yes, I just upload the video. Thanks~

**Comment by jiwansvmx** [comment]
@shupingchu Is there to only have one of those buttons in iPad ?

**Comment by shupingchu** [comment]
@jiwansvmx this issue I was trying to solve in the past few days and did quite a bit research, I couldn't find the way to access that navigationBar from `QLPreviewController` since there is no direct access api to access that navigation bar and customize it. I was gonna discuss with you if you have any idea for the workaround or how to access that layer of navigation bar. But, as I know the default share icon is sitting on the different layer of the navigation bar which is different from the customize one. I had a solution for adding another customize navigationBar with customize done button and shared icon button to cover that default one, and once preview the doc, it will show and hidden when tap on the doc in preview mode. But seems this way is kind bad performance and overhead since I need to loop each layer to find the correct one on every `viewWillAppear()` by override, and also need to set the navigation bar width and heigh base on each different devices.

**Comment by jiwansvmx** [comment]
nice 👍

**Pull request merged**